pkg/flagset/login.go

```diff
@@ -0,0 +1,54 @@
+package flagset
+
+import (
+	"github.com/micro/cli"
+	"github.com/owncloud/ocis/pkg/config"
+)
+
+// LoginWithConfig applies cfg to the root flagset
+func LoginWithConfig(cfg *config.Config) []cli.Flag {
+	return []cli.Flag{
+		&cli.StringFlag{
+			Name:        "client-id",
+			Value:       "cli",
+			Usage:       "oidc cli client id",
+			EnvVar:      "OIDC_CLI_CLIENT_ID",
+			Destination: &cfg.OIDC.ClientID,
+		},
+		&cli.StringFlag{
+			Name:        "secret",
+			Value:       "foobar",
```

Comment:
is foobar a reasonable value we want to display to the user?

Reply:
no, a proper secret would not be checked in. we can change it to smth like `idiots-don't-change-me`. make a proposal

Reply:
empty string then?

Reply:
oidc clients don't work with '' as secret. some libs require them and check for emptystring ...